lighthouse-core/test/config/config-test.js

```diff
@@ -1161,8 +1161,21 @@ describe('Config', () => {
       };
       const extendedConfig = new Config(extended);
 
+      // When gatherers have instance properties that are bind()'d, they'll not match.
+      // We'll still continue to diff the constructor via .implementation
```

Comment:
```suggestion
      // Gatherers in each config will still be compared via the constructor on .implementation.
```
the original reads a bit like we're going to diff the constructor source code (maybe that's just me reading it like that though :)